src/loops/simple_programs_with_repitition/easy/q2/VirtualBouncer.java

```diff
@@ -0,0 +1,24 @@
+package loops.simple_programs_with_repitition.easy.q2;
+
+/*
+Write a program that asks the user to enter either 1, 2, 3 , or 4, and keep asking them to enter a valid value
+until they enter one of these 4 values.
+ */
```

Comment:
I think this should have a sample output just to show that the message is printed multiple times (I know the question says this, but I think it better to include it than not).